pkg/ebpf/net_events.go

```diff
@@ -186,14 +186,14 @@ func netPacketProtocolHandler(buffer []byte, evtMeta EventMeta, ctx procinfo.Pro
 	return evt, nil
 }
 
-func parseCaptureData(payload []byte) (CaptureData, error) {
+func parseCaptureData(payload []byte) (CaptureData, int, error) {
```

Comment:
Can you just add a comment here that the int represents the packet start offset?